Source/Core/DolphinQt2/CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
-find_package(Qt5Widgets)
-find_package(Qt5Gui)
+find_package(Qt5 COMPONENTS Gui Widgets)
```

Comment:
Any thoughts on setting REQUIRED here? Or do we prefer to print our own error message?

Reply:
Personally, I'm of the opinion that required libraries should always use `REQUIRED` (it self-documents hard requirements at the point of introduction, as well as binding the failure said introduction). Though I believe the custom message was added to make it more user friendly (which I'm still skeptical about, since this actually hides the failure message that tells you which variable to set if CMake can't automatically find Qt's install path. This makes the case where you actually want to build the Qt frontend more confusing to a new person since this only tells you how to not build it).

Ideally we would either explicitly document the options in the build instructions or tell people to use `cmake -LAH` (display exposed variables with help descriptions), however we don't mark a lot of cmake variables we make as internal or advanced (which we should be doing; another thing to fix), so this would produce a lot of noise currently

That said, if others agree with the above, I can add in the `REQUIRED` and remove the error message.

Reply:
Might as well add it now since it builds (and thus requires) Qt by default. We'll have to do it anyways once we reach feature parity with WX and made it the primary UI.